test/framework/kubernetes.go

```diff
@@ -69,6 +70,7 @@ func DeleteNamespace(namespace string) error {
 	if err != nil {
 		return fmt.Errorf("Cannot delete namespace %s: %v", namespace, err)
 	}
+	OnNamespacePostDeleted(namespace)
```

Comment:
we could also migrate the `CreateNamespace` and `DeleteNamespace` methods to `namespace.go` ?
wdyt ?

Reply:
if following comment from Karel (https://github.com/kiegroup/kogito-cloud-operator/pull/349#discussion_r425678478), then this is useless ;)